### DISCUSSION TITLE
opensnitch with Smb4k

### DISCUSSION THREAD

**Discussion by smithandreas** [evilsocket/opensnitch#662] [Q&A]
Dos some one use Smb4k and open snitch? I cant get it to work that i can use my smb shares when snitch is running.

**Comment by gustavo-iniguez-goya** [comment]
hi @smithandreas , could you tell us what opensnitch version are you using? and your distro.

If you are using the package from AUR, be sure to have installed [opensnitch-ebpf-module](https://aur.archlinux.org/packages/opensnitch-ebpf-module-git)

If smb4k still doesn't work, then describe the problem in detail + set logs level to DEBUG (Preferences->Nodes) , and post the file /var/log/opensnitchd.log

**Comment by smithandreas** [comment]
Thx for your help, without any more information's, you had the right suggestion. With that AUR modul now it works (on Manjaro and with Version 1.4.2).

**Reply by gustavo-iniguez-goya** [reply]
It's a recurring question :) I should update the FAQs.
